src/Build/Collections/PropertyDictionary.cs

```diff
@@ -82,6 +84,15 @@ internal PropertyDictionary(IEnumerable<T> elements)
         }
 
         /// <summary>
+        /// Creates empty dictionary, specifying a comparer
+        /// </summary>
+        internal PropertyDictionary(MSBuildNameIgnoreCaseComparer comparer)
```

Comment:
Ah. In the tests, to assert things on the comparer. The same commit has the test changes.